nix/cardano-services/deployments/default.nix

```diff
@@ -822,10 +834,24 @@ in
             backend = {
               enabled = true;
             };
+            handle-provider.enabled = true;
             chain-history-provider.enabled = true;
+            stake-pool-provider = {
+              enabled = true;
+              env.OVERRIDE_FUZZY_OPTIONS = "true";
+            };
           };
 
+          projectors = {
+            wallet-api.enabled = true;
+            handle.enabled = true;
+            stake-pool.enabled = true;
+          };
+
+
           values = {
+            pg-boss-worker.enabled = true;
+            pg-boss-worker.queues = "pool-metadata,pool-metrics";
```

Comment:
This section seems to be intended to align the env with other env, the only line actually relevant to new projection is `wallet-api.enabled = true;`; if that's true I'm fine, otherwhise I would need some more clarifications.

In any case, the proper value for last line is `pool-delist-schedule,pool-metadata,pool-metrics,pool-rewards`.

Reply:
Fixed thank you